### PR TITLE
Update credential-guard.md

### PR DESCRIPTION
**Pull request by pyroDmanipulator**
Hello, based on issue 10257, the links have been corrected. However, they only appear based on incomplete/incorrect search results related to mitigating credential thefts. So I thought we could add the links under this content source as related topics or completely replace the other links (I submitted a new pull request for this)

### CODE REVIEW

File: windows/security/identity-protection/credential-guard/credential-guard.md [hunk 1]
```diff
@@ -43,3 +43,9 @@ By enabling Windows Defender Credential Guard, the following features and soluti
 - [What's New in Kerberos Authentication for Windows Server 2012](/previous-versions/windows/it-pro/windows-server-2012-R2-and-2012/hh831747(v=ws.11))
 - [Authentication Mechanism Assurance for AD DS in Windows Server 2008 R2 Step-by-Step Guide](/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/dd378897(v=ws.10))
 - [Trusted Platform Module](/windows/device-security/tpm/trusted-platform-module-overview)
+- [Mitigating Credential Theft using the Windows 10 Isolated User Mode](https://docs.microsoft.com/en-us/shows/seth-juarez/mitigating-credential-theft-using-windows-10-isolated-user-mode)
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/credential-guard/credential-guard.md:46]:
```suggestion
- [Mitigating Credential Theft using the Windows 10 Isolated User Mode](https://docs.microsoft.com/en-us/shows/seth-juarez/mitigating-credential-theft-using-windows-10-isolated-user-mode)
```

File: windows/security/identity-protection/credential-guard/credential-guard.md [hunk 2]
```diff
@@ -43,3 +43,9 @@ By enabling Windows Defender Credential Guard, the following features and soluti
...
+- [Isolated User Mode Processes and Features in Windows 10 with Logan Gabriel](https://docs.microsoft.com/en-us/shows/seth-juarez/isolated-user-mode-processes-features-in-windows-10-logan-gabriel)
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/credential-guard/credential-guard.md:47]:
```suggestion
- [Isolated User Mode Processes and Features in Windows 10 with Logan Gabriel](https://docs.microsoft.com/en-us/shows/seth-juarez/isolated-user-mode-processes-features-in-windows-10-logan-gabriel)
```

File: windows/security/identity-protection/credential-guard/credential-guard.md [hunk 3]
```diff
@@ -43,3 +43,9 @@ By enabling Windows Defender Credential Guard, the following features and soluti
...
+- [More on Processes and Features in Windows 10 Isolated User Mode with Dave Probert](https://docs.microsoft.com/en-us/shows/seth-juarez/more-on-processes-features-in-windows-10-isolated-user-mode-dave-probert)
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/credential-guard/credential-guard.md:48]:
```suggestion
- [More on Processes and Features in Windows 10 Isolated User Mode with Dave Probert](/shows/seth-juarez/more-on-processes-features-in-windows-10-isolated-user-mode-dave-probert)
```

File: windows/security/identity-protection/credential-guard/credential-guard.md (hunk 1 above)
Comment (JohanFreelancer9) [inline windows/security/identity-protection/credential-guard/credential-guard.md:46]:
```suggestion
- [Mitigating Credential Theft using the Windows 10 Isolated User Mode](/shows/seth-juarez/mitigating-credential-theft-using-windows-10-isolated-user-mode)
```

File: windows/security/identity-protection/credential-guard/credential-guard.md (hunk 2 above)
Comment (JohanFreelancer9) [inline windows/security/identity-protection/credential-guard/credential-guard.md:47]:
```suggestion
- [Isolated User Mode Processes and Features in Windows 10 with Logan Gabriel](/shows/seth-juarez/isolated-user-mode-processes-features-in-windows-10-logan-gabriel)
```

File: windows/security/identity-protection/credential-guard/credential-guard.md (hunk 3 above)
Comment (JohanFreelancer9) [inline windows/security/identity-protection/credential-guard/credential-guard.md:48]:
```suggestion
- [More on Processes and Features in Windows 10 Isolated User Mode with Dave Probert](/shows/seth-juarez/more-on-processes-features-in-windows-10-isolated-user-mode-dave-probert)
```

File: windows/security/identity-protection/credential-guard/credential-guard.md [hunk 4]
```diff
@@ -43,3 +43,9 @@ By enabling Windows Defender Credential Guard, the following features and soluti
...
+- [Isolated User Mode in Windows 10 with Dave Probert](https://docs.microsoft.com/en-us/shows/seth-juarez/isolated-user-mode-in-windows-10-dave-probert)
+- [Windows 10 Virtual Secure Mode with David Hepkin](https://docs.microsoft.com/en-us/shows/seth-juarez/windows-10-virtual-secure-mode-david-hepkin)
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/credential-guard/credential-guard.md:50]:
```suggestion
- [Windows 10 Virtual Secure Mode with David Hepkin](/shows/seth-juarez/windows-10-virtual-secure-mode-david-hepkin)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @pyroDmanipulator, kindly commit the suggested changes. Many thanks,

**Comment by pyroDmanipulator** [comment]
> Copy review - @pyroDmanipulator, kindly commit the suggested changes. Many thanks,

I have made another pull request with the needed corrections. Kindly review 
Thank you, have a nice day

**Comment by JohanFreelancer9** [comment]
@pyroDmanipulator, I'll do a copy edit on the new pull request and close this one. Thanks.

**Pull request closed without merging**